6.X/6.9.0/docs/03-upgrade.md

```diff
@@ -0,0 +1,125 @@
+<!-- Space: TOOLS -->
+<!-- Parent: Installer -->
+<!-- Title: Upgrade -->
+<!-- Layout: plain -->
+
+# Upgrade
+
+<br />
+
+<!-- Include: ac:toc -->
+
+<br />
+
+## Overview
+
+The Installer can be used to upgrade a Sysdig implementation. As in an
+install, you must meet the prerequisites, download the values.yaml, edit the
+values as indicated, and run the Installer. The main difference is that you
+run it twice: once to discover the differences between the old and new
+versions, and the second time to deploy the new version.
+
+As with installs, it can be used in airgapped or non-airgapped environments.
+
+Review the [Prerequisites](../README.md#prerequisites) and [Installation
+Options](../README.md#quickstart-install) for more context.
+
+## Upgrade Steps
+
+<br />
+
+### Step 1
```

Comment:
Please use a heading that is intuitive. I remember giving the suggestion last time as well. `Step` does not reveal anything about the steps you are going to do.